services/event/models/checkin_result.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type CheckinResult struct {
+	NewPoints   int    `json:"newPoints"`
```

Comment:
See my comment about lowercase camelcase above.

Reply:
The variable name of the struct should be uppercase camel case, but the json should be lower camel case (this is consistent with the casing found in the other models, i.e. the event.go model).

Reply:
Gotcha, just wanted to double check